Modules/TPC/include/TPC/PadCalibrationCheck.h

```diff
@@ -46,6 +46,11 @@ class PadCalibrationCheck : public o2::quality_control::checker::CheckInterface
   std::vector<float> mNoiseMean;
   std::vector<float> mNoiseStdDev;
   std::vector<int> mNoiseNonZeroEntries;
+  std::vector<std::string> mMOsToCheck2D;
+  double mMediumQualityLimit;
+  double mBadQualityLimit;
+  double mMediumQualityLimitNoiseMean;
+  double mBadQualityLimitNoiseMean;
```

Comment:
I think the idea was to generalize the class to be able to use it for all PadCalibration objects. So I would assume that no specific reference to e.g. `Noise` should appear.